Applications/IK/test/constraintTest_ikOnly.osim

```diff
@@ -5941,7 +5941,7 @@
 						<groups/>
 					</WrapObjectSet>
 				</Body>
-				<Body name="toes_l">
+				<Body name="toes_left">
 					<mass>       0.09750000 </mass>
```

Comment:
That is why I had switched the procedure of making a connection to use `Connector<C>::findAndConnect()` which does filter on Component type. I think we should understand why that wasn't working for you in another instance, because it was certainly working in this case.